core/chain_makers.go

```diff
@@ -352,6 +369,13 @@ func GenerateChain(config *params.ChainConfig, parent *types.Block, engine conse
 		if err != nil {
 			panic(err)
 		}
+		if config.IsCancun(block.Number(), block.Time()) {
```

Comment:
https://github.com/bnb-chain/bsc/blob/v1.4.15/core/chain_makers.go#L375-L377
```suggestion
		if block.Header().EmptyWithdrawalsHash() {
			block = block.WithWithdrawals(make([]*types.Withdrawal, 0))
		}
		if config.IsCancun(block.Number(), block.Time()) {
```